<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixesWidget</class>
 <widget class="QWidget" name="FixesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FixesWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="hpMpBarsSizeBox">
     <property name="title">
      <string>Hp Mp Bars Size</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="hpMpBarsSizeSlider">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="maximum">
         <number>47</number>
        </property>
        <property name="value">
         <number>47</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hpMpBarsSizeValue">
        <property name="text">
         <string>47</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="specialAttackModifiersDisplayBox">
     <property name="title">
      <string>Special Attack Modifiers Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="specialAttackModifiersDisplayFix">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Fix</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="bodyEnhancementBox">
     <property name="title">
      <string>Body Enhancement</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="bodyEnhancementFix">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Fix</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="autumnMoonEffectBox">
     <property name="title">
      <string>Autumn Moon Effect</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="autumnMoonEffectFix">
        <property name="text">
         <string>Fix</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
